client/app/index.js

```diff
@@ -13,8 +13,7 @@ ngModule.config(($locationProvider, $compileProvider, uiSelectConfig, toastrConf
 });
 
 // Update ui-select's template to use Font-Awesome instead of glyphicon.
-// eslint-disable-next-line no-unused-vars
-ngModule.run(($templateCache, OfflineListener) => {
```

Comment:
Now that you removed `OfflineListener` here you need to explicitly import it.

Reply:
No (actually, I tested it - it works :smile:) - it's inside init() function and is automatically invoked on application startup

Reply:
Oh, right. I forgot about my own magic... ✨